src/core/constants/sacd.ts

```diff
@@ -0,0 +1,4 @@
+// SACD Permissions
+// Binary: 1111111111 --> Decimal: 1023
```

Comment:
I believe we have 6 permissions https://github.com/DIMO-Network/web3-deploys/blob/main/Polygon/PolygonDeploys.md

Also, I don't think this is set in stone, but I believe we want to mimic the way the privilege system works. So, we would start on index 1 instead of 0. Sounds like a waste, let's discuss